pkg/users/account-details.js

```diff
@@ -164,16 +175,14 @@
                 .catch(show_unexpected_error);
     }
 
-    if (!accounts.length) {
+    if (isLoading) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-5e987a2f-20250813-131607-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)